components/claim/components/email_to_ldap.tsx

```diff
@@ -146,11 +159,12 @@ export default class EmailToLDAP extends React.PureComponent {
         }
 
         let content;
+        var defaultState = this.getDefaultState()
         if (this.state.showMfa) {
             content = (
                 <LoginMfa
                     loginId={this.props.email}
-                    password={this.state.password}
+                    password={defaultState.password}
```

Comment:
why storing password in the state (there is even an `ldappassword` field in it if we are always passing `''`?

Reply:
so just using this.state.password not referring to defaultState?

Reply:
nevermind this, I think I misread something